spec/models/pacbio/run_spec.rb

```diff
@@ -154,7 +155,12 @@
       run.update(name: 'run1')
       expect(run.name).to eq('run1')
     end
-
   end
 
+  context 'traction_id' do
+    it 'should combine TRACTION- with the run id' do
+      runs = create_list(:pacbio_run, 3)
+      expect(runs.map(&:traction_id)).to match_array(runs.map { |r| "TRACTION-#{r.id}" })
+    end
+  end
```

Comment:
This is the test I added - everything else here was to satisfy Rubocop during the commit